packages/suite/src/views/onboarding/steps/Final/index.tsx

```diff
@@ -230,30 +235,45 @@ export const FinalStep = () => {
                         </SetupActions>
                     )}
                     {state === 'rename' && (
-                        <SetupActions>
-                            <Input
-                                noTopLabel
-                                noError
-                                value={label}
-                                placeholder={DEFAULT_LABEL}
-                                inputState={label.length > MAX_LABEL_LENGTH ? 'error' : undefined}
-                                onChange={(event: React.FormEvent<HTMLInputElement>) =>
-                                    setLabel(event.currentTarget.value)
-                                }
-                                data-test="@settings/device/label-input"
-                            />
-                            <Button
-                                onClick={async () => {
-                                    await onRename();
-                                }}
-                                isDisabled={
-                                    isDeviceLocked || !label || label.length > MAX_LABEL_LENGTH
-                                }
-                                data-test="@settings/device/label-submit"
-                            >
-                                <Translation id="TR_DEVICE_SETTINGS_DEVICE_EDIT_LABEL" />
-                            </Button>
-                        </SetupActions>
+                        <>
+                            <Description>
```

Comment:
It's ugly, I would show it only in case of error like that
![Screenshot 2023-06-27 at 15 18 01](https://github.com/trezor/trezor-suite/assets/33235762/bec6d858-5a3b-4eee-b202-daed04186d14)

I think it would be best change this step anyway @Hannsek [#8702](https://github.com/trezor/trezor-suite/issues/8702#issuecomment-1593049688)

Reply:
Which step exactly?